source_files/gpu.c

```diff
@@ -35,22 +42,20 @@
 #include "gpu.h"
 
 #define CUDA_ERROR_CHECK
-#define CudaSafeCall( err ) __cudaSafeCall( err, __FILE__, __LINE__ )
-#define CudaCheckError() __cudaCheckError( __FILE__, __LINE__ )
+#define cuda_safe_call( cuerr ) __cuda_safe_call( cuerr, __FILE__, __LINE__ )
 
-static void *A,*B;
```

Comment:
These are used later. E.g., cuMemcpyHtoD_v2(d_Adata, A, d_resultSize) Either they have to stay or the call has to go. I assume the latter.